packages/property-projector/src/components/ui/config-form/department-checkboxes.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { FormSection, FormHeading, FormSubHeading, Checkbox } from '@reapit/elements'
+
+type DepartmentCheckboxesProps = {
+  departments: any[]
```

Comment:
Try and find definition in `@reapit/foundations-ts-definitions` so departments array is strongly typed

Reply:
Very likely to be `DepartmentModel[]`

Reply:
https://github.com/reapit/foundations/blob/master/packages/foundations-ts-definitions/types/platform-schema.ts#L4186